caom2/caom2/chunk.py

```diff
@@ -82,31 +82,57 @@
 
 class ProductType(OrderedEnum):
     """
-    SCIENCE: "science"
-    CALIBRATION: "calibration"
-    PREVIEW: "preview"
-    INFO: "info"
-    NOISE: "noise"
-    WEIGHT: "weight"
-    AUXILIARY: "auxiliary"
-    THUMBNAIL: "thumbnail"
-    BIAS: "bias"
-    DARK: "dark"
-    FLAT: "flat"
-    WAVECAL: "wavecal"
+    Subset of IVOA DataLink terms
```

Comment:
Would it be useful to add a link to the page with the origin of these terms: e.g. to https://www.ivoa.net/rdf/datalink/core/2022-01-27/datalink.html